hphp/runtime/vm/jit/vasm-ppc64.cpp

```diff
@@ -474,6 +474,11 @@ struct Vgen {
 private:
   CodeBlock& frozen() { return text.frozen().code; }
 
+  void emitSaveTOC() {
+    // TOC save/restore is required by ABI for external functions.
+    a.std(rtoc(), rsfp()[AROFF(SAVED_TOC())]);
```

Comment:
This function is called in some other places, but as this change was related with the backchain, and it is used in the `inittc`, I called it only there. There will be another PR to add the other calls. I think naming a function is a better description of what the asm line is doing.